jboss-kie-wildfly-common/tests/bats/jboss-kie-wildfly-common.bats

```diff
@@ -27,6 +27,27 @@ teardown() {
     rm -rf $JBOSS_HOME
 }
 
+setupFilesAndFoldersForCleanupTests(){
+      mkdir -p  ${JBOSS_HOME}/domain
+      mkdir -p  ${JBOSS_HOME}/migration
+      mkdir -p  ${JBOSS_HOME}/.installation
+      mkdir -p  ${JBOSS_HOME}/bin/init.d/
+      touch ${JBOSS_HOME}/bin/test.bat
+      touch ${JBOSS_HOME}/bin/test.ps1
+      touch ${JBOSS_HOME}/bin/test.sh
+      touch ${JBOSS_HOME}/bin/init.d/test.sh
+      mkdir -p  ${JBOSS_HOME}/standalone/configuration/
+      touch ${JBOSS_HOME}/standalone/configuration/standalone-full.xml
+      touch ${JBOSS_HOME}/standalone/configuration/standalone-ha.xml
+      touch ${JBOSS_HOME}/standalone/configuration/standalone-full-ha.xml
+      touch ${JBOSS_HOME}/standalone/configuration/standalone-load-balancer.xml
```

Comment:
Sure @jakubschwan. 
Do you think that it could be a good idea also adding a check in the [behave test](https://github.com/davidesalerno/jboss-kie-modules/blob/a9bd1d7c9221f40d345e73724ba890f56d492dfc/tests/features/common/kie-common.feature#L363)?

Reply:
yes, sure. I noticed it's already added :+1: